Segunda_parte/ejercicio13.c

```diff
@@ -0,0 +1,78 @@
+#include <stdio.h>
+#include <string.h>
+
+/*
+Escribir un programa que valide si una dirección de correo electrónico 
+es válida. Para considerarla válida, debe tener al menos tres 
+caracteres, seguidos de un “@”, seguido de al menos dos letras o 
+números (sin caracteres especiales), luego un punto y dos caracteres al 
+final. Ejemplo válido: ejemplo@gmail.com , Ejemplo válido: aa@!!.co 
+*/
+
+int main () {
+	char str1[100];
+	int i = 0, Validate3Letters = 1, Validate2Letters = 1, NumLetters = 0;
+	
+	printf("Ingrese el correo a validar:\n");
+	fgets(str1, sizeof(str1), stdin);
+	
+	if (strlen(str1) >= 10){
+		while (str1[i] != '\0'){
+			for (; i <3; i++){
+				if ((str1[i] >= 65 && str1[i] <= 90) || (str1[i] >= 97 && str1[i] <= 122)){
+					
+				}else{
```

Comment:
tener un bloque `if` vacio se considera una mala practica. En su lugar tu condicion del `if` puede pasar a ser la negacion (`!` not operator) de la que escribiste asi no necesitas un `if` vacio ni un `else`